app/scripts/views/sign_in.js

```diff
@@ -95,6 +96,12 @@ define(function (require, exports, module) {
       this._formPrefill.set('password', this.getElementValue('.password'));
     },
 
+    hideSuccessAndErrorMessages: function () {
+      $(this.el).off('keypress', '.input-row');
```

Comment:
> 1) would you like me to leave it bound for every key stroke?
> 2) Sorry, but I didn't understand why it's better to move the event to FormView. 

Yeap, no worries. Several goals / benefits of putting it into `FormView` :
 - fix this for all possible forms, we want the default behaviour on change to hide success / error messages. 
 - avoid UX surprises, where different forms do different things with error messages
 - we already have event handlers in FormView that we can use to achieve our goal.

I just checked again and we already have listeners in place: https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/form.js#L61  

I think you can just make a change: https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/form.js#L116

Old: 
```
      if (this.isValid()) {
        this.hideError();
        this.enableForm();
      } else {
        this.disableForm();
      }
```

New: 
```
      this.hideError();
      this.hideSuccess();
      if (this.isValid()) {
        this.enableForm();
      } else {
        this.disableForm();
      }
```
Try it out!

And then just update/add some tests. Let me know if you have questions